web/src/components/AppContainer.jsx

```diff
@@ -31,6 +31,10 @@ export default function AppContainer({children}) {
         <link rel="mask-icon" href="/safari-pinned-tab.svg" color="#5bbad5" />
         <meta name="msapplication-TileColor" content="#da532c" />
         <meta name="theme-color" content="#ffffff" />
+        <meta
+          name="description"
+          content="Get motivated to develop for Web3 and build an NFT marketplace!"
```

Comment:
```suggestion
          content="Get motivated to develop for web3 and build an NFT marketplace!"
```
Not sure what our style guide says for "web3" but I always preferred lowercase w 🤷